src/puppeteer_pool.js

```diff
@@ -266,25 +273,7 @@ class PuppeteerPool {
         const id = this.browserCounter++;
         log.debug('PuppeteerPool: Launching new browser', { id });
 
-        const errorMessageChunk = 'launchPuppeteerFunction timed out.';
-        const launchPromise = this.launchPuppeteerFunction();
-        const browserPromise = addTimeoutToPromise(
```

Comment:
Is it okay to completely remove this functionality?

Reply:
I replaced it with the `puppeteer.launch({ timeout })` option, so I guess we'll be better off because this way we don't have to handle killing of the browser.